react/DeliveryItems.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react'
+
+import ProductList from './components/ProductList'
+import ShippingHeader from './components/DeliveryHeader'
+import { useOrder } from './components/OrderContext'
+import OrderSection from './OrderSection'
+
+const Shipping: FC = () => {
+  const { deliveryParcels, giftRegistryData } = useOrder()
+
+  if (deliveryParcels.length === 0) {
+    return null
+  }
+
+  return (
+    <OrderSection>
+      {deliveryParcels.map((deliveryParcel, index) => (
+        <div
+          className="mv8 flex-l justify-between flex-column flex-row-m"
+          key={index}
```

Comment:
Not necessarily. I'm not reordering, adding or removing any items from this array. If I was manipulating it somewhere, sure, it would be an anti-pattern. 

See the `Update: Exception from the rule` section on: https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
1. If your list is static and will not change. ✅ 
2. The list will never be re-ordered.  ✅ 
3. The list will not be filtered (adding/removing items from the list).  ✅ 
4. There are no ids for the items in the list.  ✅

Reply:
all right